### DISCUSSION TITLE
Migration Guide to Pimcore X/10

### DISCUSSION THREAD

**Discussion by M-Ali-Aziz** [pimcore/pimcore#7979] [Q&A / Support] [answered]
Hi,
I was wondering when Pimcore is planing to provide a **Migration/Upgrade guide from Pimcore 6 to 10** in the documentation page?

Regards
Ali

**Comment by brusch** [comment]
Well, 10 is not finished yet, so it will be there  latest when 10 is released (at least as a beta/RC) 😊

**Reply by M-Ali-Aziz** [reply]
Thanks for your reply :)
This migration will be very time consuming and I was wondering if we could start now and prepare for the update to Pimcore 10.

I understand what you mean but would be grateful if I could get some advice on what it is we can start right now so that we can more easily and faster update to Pimcore 10.

For example, we know that we need to start rewriting the php-template to twig.

I was wondering if we could get more advice on what more do we need to think about and start planning for?

**Reply by dpfaffenbauer** [reply]
Most of it comes from Symfony, so check there upgrade guide to Symfony 5....

**Reply by M-Ali-Aziz** [reply]
Thank you Dominik! :)
I'll check it.

**Comment by M-Ali-Aziz** [comment, marked as answer]
Upgrading Pimcore from Version 6.x to Version 10
https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/Updating_Pimcore/V6_to_V10.html

Thank you :)
